docs/esp32/quickref.rst

```diff
@@ -102,6 +106,7 @@ Once the network is established the :mod:`socket <usocket>` module can be used
 to create and use TCP/UDP sockets as usual, and the ``urequests`` module for
 convenient HTTP requests.
 
+
```

Comment:
This additional whiteline seems unrelated to the rest of the commit. Was it done by mistake?

Reply:
Is a mistake, sure. However, this PR is most likely obsoleted by #7348.